### ISSUE TITLE
Viewing/Editing Metabase Docker Container H2 Database

### ISSUE THREAD

**Issue by wolfem4n** [metabase/metabase#7180]
Hi there. Sorry in advance, I'm a novice in regards to managing metabase instances.

We have an issue similar to #5842 & #4936, where we have removed a user(Created with Google SSO) from the metabase GUI, and now that user is unable to access metabase again using google SSO. He gets a similar unauthenticated error as mentioned in #5842.

I am trying to view/edit the h2 application database(metabase.db.mv.db file) in order to restore the active flag for the particular user that is having issues. I can't seem to find any tools to allow me to view/edit the H2 database, or some of them will say the db is locked or encrypted(no encryption key, or passwords have been set on the db). I'm assuming that is because I'm using the wrong tools to try and view the metabase.db.mv.db file. 

Our metabase instance is run on a docker container, with the database being mounted outside the container for persistent storage: 

docker run -d -p 3000:3000 -v ~/metabase-data:/metabase-data -e "MB_DB_FILE=/metabase-data/metabase.db" --name metabase metabase/metabase:latest

I was finally able to open a backup of the database file, using H2 Console, but all I was able to see was a generic information schema, and no relevant data from our metabase configuration or users. I've re-run the docker container using this database, so I know it contains information about our metabase config and users.

Our server is running CentOS 7. Any help or advice on how I can view or edit the H2 database, inside or outside of the container, would be much appreciated. I feel like I may be missing something obvious.

Thanks very much.

**Comment by jornh** [comment]
I would recommend connection outside the container (and shutting Metabase down before + make sure you have a backup).

Which connection string are you using? It should end in `metabase.db` *not* `Metabase.db.mv.db`. See [the two H2 GUI screenshots in this Discourse](http://discourse.metabase.com/t/migrate-from-h2-to-postgresql/597/3?u=jornh) topic

**Comment by wolfem4n** [comment]
@jornh appreciate you looking into this. The database file I am trying to view is the backup copy of our metabase db mounted on our server. I looked at the two screenshots you had linked, and the connection string I was using was pointing to the metabase.db.mv.db file. I renamed the database file to metabase.db, instead of metabase.db.mv.db. However, even after using the same connection strings as shown in those screenshots, it doesn't show any relevant tables, and still only shows a generic information schema template. 

![screen shot 2018-03-23 at 4 36 14 pm](https://user-images.githubusercontent.com/30910340/37854340-8bd0984e-2eb8-11e8-9dc1-96cb014f0a7e.png)

![screen shot 2018-03-23 at 4 36 39 pm](https://user-images.githubusercontent.com/30910340/37854358-944bd092-2eb8-11e8-9909-be60cbf94b7d.png)

Before when I was using the wrong connection string, H2 console would create another blank metabase.db.mv.db file, and I could tell that the connection string was incorrect. But even after using the correct string, and pointing to the correct file, I'm still only seeing a generic information schema. 

Any other ideas would be appreciated. Will continue to do some testing this weekend. 

Thanks again.
